src/dreyfus_index_manager.erl

```diff
@@ -86,6 +86,12 @@ handle_call({open_error, DbName, Sig, Error}, {OpenerPid, _}, State) ->
 
 handle_cast({cleanup, DbName}, State) ->
     clouseau_rpc:cleanup(DbName),
+    {noreply, State};
+
+handle_cast({move, DbName}, State) ->
```

Comment:
Yes, I run a successful move before having this commit. It looks that `gen_server:cast(?MODULE, {move, DbName})` matches with `handle_cast({move, DbName}, State)`.